Cannot find module 'function () ...

When using dynamic import syntax `import()`, the following error can sometime occur:
```
Unhandled promise rejection (rejection id: 1): Error: Cannot find module 'function () {
  return require('...');
}'
```

--- a/src/packages/babel-preset-dev-toolkit/index.js
+++ b/src/packages/babel-preset-dev-toolkit/index.js
@@ -19,22 +19,30 @@ if (env !== 'development' && env !== 'test' && env !== 'production') {
 module.exports = {
   presets: [
     require.resolve('babel-preset-env'),
-    // Use create-react-app default which also supports dynamic `import()`-statement
+    // Use create-react-app default
     require.resolve('babel-preset-react-app'),
   ],
-  plugins: [
-    // Sane if-statements for React
-    require.resolve('jsx-control-statements'),
-    // Allow root-relative imports for client & server
-    [
-      require.resolve('babel-plugin-module-resolver'),
-      {
-        // using `process.cwd` makes it also work with `import()`
-        root: ['./src'],
-        alias: {
-          src: path.resolve(process.cwd(), 'src'),
+  plugins: []
+    .concat(
+      env !== 'test'
+        ? // Support dynamic `import()`-statement everywhere
+          [require.resolve('babel-plugin-transform-dynamic-import')]
+        : // Ignore transform-dynamic-import in `test` env since `babel-preset-react-app` already includes it
+          []
+    )
+    .concat([
+      // Sane if-statements for React
+      require.resolve('jsx-control-statements'),
+      // Allow root-relative imports for client & server
+      [
+        require.resolve('babel-plugin-module-resolver'),
+        {
+          // using `process.cwd` makes it also work with `import()`
+          root: ['./src'],
+          alias: {
+            src: path.resolve(process.cwd(), 'src'),
+          },
         },
-      },
-    ],
-  ],
+      ],
+    ]),
 };

--- a/src/packages/babel-preset-dev-toolkit/index.js
+++ b/src/packages/babel-preset-dev-toolkit/index.js
@@ -19,12 +19,10 @@ if (env !== 'development' && env !== 'test' && env !== 'production') {
 module.exports = {
   presets: [
     require.resolve('babel-preset-env'),
-    // Use create-react-app default
+    // Use create-react-app default which also supports dynamic `import()`-statement
     require.resolve('babel-preset-react-app'),
   ],
   plugins: [
-    // Support dynamic `import()`-statement
-    require.resolve('babel-plugin-dynamic-import-node'),
     // Sane if-statements for React
     require.resolve('jsx-control-statements'),
     // Allow root-relative imports for client & server

--- a/src/packages/babel-preset-dev-toolkit/package.json
+++ b/src/packages/babel-preset-dev-toolkit/package.json
@@ -1,6 +1,6 @@
 {
   "name": "babel-preset-dev-toolkit",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "description": "Uses create-react-app default, adds sane if-statements & root-relative imports",
   "main": "index.js",
   "scripts": {
@@ -29,7 +29,6 @@
   "author": "Jorge Antunes @stoikerty",
   "license": "MIT",
   "dependencies": {
-    "babel-plugin-dynamic-import-node": "^1.2.0",
     "babel-plugin-module-resolver": "^3.1.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react-app": "^3.1.1",

--- a/src/packages/babel-preset-dev-toolkit/package.json
+++ b/src/packages/babel-preset-dev-toolkit/package.json
@@ -1,6 +1,6 @@
 {
   "name": "babel-preset-dev-toolkit",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "description": "Uses create-react-app default, adds sane if-statements & root-relative imports",
   "main": "index.js",
   "scripts": {
@@ -30,6 +30,7 @@
   "license": "MIT",
   "dependencies": {
     "babel-plugin-module-resolver": "^3.1.0",
+    "babel-plugin-transform-dynamic-import": "^2.0.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react-app": "^3.1.1",
     "jsx-control-statements": "^3.2.8"

